### PR TITLE
installation on JSC infrastructure

### PR DESCRIPTION
**Pull request by adswa**
This adds installation and configuration instructions for the JSC to the installation section of the INM-7 version of the handbook.

### CODE REVIEW

**Comment by bpoldrack** [comment]
Not an actual review yet, but an observation for now as I try to set this up myself:

While datalad 0.12.5 is installed as expected including git-annex-remote-datalad (-archive) special remotes, `git-annex-remote-ora` entry point is missing.

**Comment by mih** [comment]
> While datalad 0.12.5 is installed as expected including git-annex-remote-datalad (-archive) special remotes, git-annex-remote-ora entry point is missing.

Not really a surprise. The 0.12 doesn't have that functionality ;-)

File: docs/index.rst [hunk 1]
```diff
@@ -17,6 +17,13 @@ The DataLad Handbook
 **Welcome to the DataLad handbook!**
 ------------------------------------
 
+.. important::
+
+   You are viewing the **INM-7-specific** version of the handbook. Most content
```
Comment (mih) [inline docs/index.rst:22]: Maybe add a link to the institute? Most people will have no idea what INM7 is.

File: docs/intro/installation.rst [hunk 2]
```diff
@@ -32,6 +29,194 @@ exist, and they need to be installed separately, if needed.
...
+        ################################################################################
+        #                           Welcome to JuDAC                                   #
+        #                          Juelich Data Access                                 #
+        #------------------------------------------------------------------------------#
+        #              Information about the system, latest changes,                   #
+        #                       user documentation and FAQs:                           #
+        #                http://www.fz-juelich.de/ias/jsc/judac                        #
+        #------------------------------------------------------------------------------#
+        ################################################################################
+        Last login: Tue Feb  4 16:36:10 2020 from bnbnb29.ime.kfa-juelich.de
```
Comment (mih) [inline docs/intro/installation.rst:63]: I think the MOTD can go. it adds nothing.

**Pull request merged**